ngraph/frontend/paddlepaddle/src/model.cpp

```diff
@@ -182,13 +182,47 @@ Place::Ptr InputModelPDPD::InputModelPDPDImpl::getPlaceByTensorName (const std::
 }
 
 void InputModelPDPD::InputModelPDPDImpl::overrideAllInputs (const std::vector<Place::Ptr>& inputs) {
-    // TODO: resolve for different kind of places
-    m_inputs = inputs;
+    m_inputs.clear();
+    for (const auto& inp : inputs) {
+        auto var_place = std::dynamic_pointer_cast<TensorPlacePDPD>(inp);
```

Comment:
you can try
if(auto var_place = std::dynamic_pointer_cast<TensorPlacePDPD>(inp) {
    m_inputs.push_back(var_place);
    continue;
}